Examples/restcomm-olympus-nopod/restcomm-olympus/MainTableViewController.h

```diff
@@ -29,7 +29,8 @@
 #import "SipSettingsTableViewController.h"
```

Comment:
3. We will implement number 2. so we don't have this case
4. I would go with option to show always search, I think it will be a little bit confusing for customer to popup search on scroll up in the middle of the list. wdyt?
5. I have used systems icon. Will check

Reply:
3. I lost you here, can you elaborate how 2. will help on 3.? 3. is about getting completely empty contact: empty username AND empty number, while 2. is about a contact with proper username but not number. Am I missing something here?
4. Agreed, let's make it prominent, so that users are instantly aware of its presence
5. Thanks

Reply:
3. If we dont have number, we will not add user to Olympus.